src/main/java/seedu/address/logic/parser/AddressBookParser.java

```diff
@@ -52,12 +54,20 @@ public Command parseCommand(String userInput) throws ParseException {
         final String commandWord = generalMatcher.group("commandWord");
         final String commandArguments = generalMatcher.group("commandArguments");
 
+        NusModulesParser nusModulesParser;
+
+        try {
+            nusModulesParser = new NusModulesParser();
+        } catch (IOException e) {
+            throw new ParseException("NUS Modules not found!");
+        }
```

Comment:
is this part still needed? I belive it was just previously used for debugging right

Reply:
No its actually there to throw an exception if its not found